README.md

```diff
@@ -17,11 +17,51 @@ Inspired by [`YousefED/typescript-json-schema`](https://github.com/YousefED/type
 
 This project is made possible by a [community of contributors](https://github.com/vega/ts-json-schema-generator/graphs/contributors). We welcome contributions of any kind (issues, code, documentation, examples, tests,...). Please read our [code of conduct](https://github.com/vega/vega/blob/master/CODE_OF_CONDUCT.md).
 
-## Usage
+## CLI Usage
 
 ```bash
 npm install --save ts-json-schema-generator
-./node_modules/.bin/ts-json-schema-generator --path 'my/project/**.*.ts' --type 'My.Type.Full.Name'
+./node_modules/.bin/ts-json-schema-generator --path 'my/project/**.*.ts' --type 'My.Type.Full.Name'  // Logs the schema to console
+./node_modules/.bin/ts-json-schema-generator --path 'my/project/**.*.ts' --type 'My.Type.Full.Name' --out 'my/output/file.json'
```

Comment:
```suggestion
./node_modules/.bin/ts-json-schema-generator --path 'my/project/**.*.ts' --type 'My.Type.Full.Name'
```

This is already documented

Reply:
- The CLI option was missing from [here](https://github.com/vega/ts-json-schema-generator#options), so added that and showed an example usage above. 
- Can remove `./node_modules/.bin/ts-json-schema-generator --path 'my/project/**.*.ts' --type 'My.Type.Full.Name' ` if thats preferred. 
- `./node_modules/.bin/ts-json-schema-generator --path 'my/project/**.*.ts' --type 'My.Type.Full.Name' --out 'my/output/file.json'` seems more useful than the other command.

Reply:
You can always pipe to a file so I don’t agree.

Reply:
Okay got it, I didn't think of piping.